ia-configs/IA.whaleshark.json

```diff
@@ -1,42 +1,35 @@
 {
-    "_note":"The hierarchy of this file is genus->species->ia_class->_id_conf (id config). detect_config is under Genus->species. Keys with a leading underscore and their children are non-biological categories of config, like _id_conf and _detection_conf. Keys without leading underscores are semantic lookups that are derived from Wildbook data, e.g. Megaptera.novaeangliae.whale_fluke.",
-    "_default":"@Rhincodon.typus",
-    "Rhincodon":{
-
-       "typus":{
-
-          "whalesharkCR":{
-
-             "_id_conf":[
-                {
-                   "query_config_dict":{
-                      "pipeline_root": "PieTwo"
-                   },
-                   "description":"PIE v2 pattern-matcher"
-                },
-                {
-                   "query_config_dict":{
-                      "sv_on":false
-                   },
-                   "description":"SLOW: HotSpotter body pattern-matcher",
-                   "default": false
-                }
-             ]
+  "Rhincodon": {
+    "typus": {
+      "_default": "@Rhincodon.typus.whaleshark",
+      "_detect_conf": [
+        {
+          "algo": "yolo",
```

Comment:
This new attribute `algo` is used to send the detection job to the older YOLO detector